### PR TITLE
List SPM as the recommended package manager

### PR DESCRIPTION
**Pull request by jhgilbert**
### What does this PR do? What is the motivation?
Fulfills a request made during a security review of [this PR](https://github.com/DataDog/documentation/pull/24142).

DOCS-8402

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by maxep — APPROVED** [review]
Thank you, @jhgilbert 🙏

**Review by olacin-dd — APPROVED** [review]
Approving for the security incident response team 👍 

Thank you !

**Review by plousada — APPROVED** [review]

File: content/en/tracing/trace_collection/custom_instrumentation/ios/otel.md [hunk 1]
```diff
@@ -48,6 +38,16 @@ DatadogCore
 DatadogTrace
 ```
 
+{{% /tab %}}
+{{% tab "CocoaPods" %}}
+
+You can use [CocoaPods](https://cocoapods.org/) to install `dd-sdk-ios`:
```
Comment (drichards-87) [inline content/en/tracing/trace_collection/custom_instrumentation/ios/otel.md:44]: Just FYI, this link is formatted inline.

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left a note about an inline link.

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**